man/systemd.netdev.xml

```diff
@@ -1220,7 +1220,19 @@
         <term><varname>DiscoverPathMTU=</varname></term>
         <listitem>
           <para>Takes a boolean. When true, enables Path MTU Discovery on
-          the tunnel.</para>
+          the tunnel.
+          When <varname>IgnoreDF=</varname> is explicitly set to true,
+          defaults to false. Otherwise, defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>IgnoreDF=</varname></term>
```

Comment:
maybe spell this out as IgnoreDontFragment=? no need to be needlessly cryptic. makes the .netdev file a lot more self-explanatory, people wouldn't have to read up the docs anymore to see wtf this is.

Reply:
Makes sense. Renamed.